config/confighttp/confighttp.go

```diff
@@ -99,6 +100,10 @@ type ClientConfig struct {
 	// HTTP2PingTimeout if there's no response to the ping within the configured value, the connection will be closed.
 	// If not set or set to 0, it defaults to 15s.
 	HTTP2PingTimeout time.Duration `mapstructure:"http2_ping_timeout"`
+
+	// CookiesEnabled enables reusing cookies between requests.
+	// This is useful to manage sticky session cookies.
+	CookiesEnabled bool `mapstructure:"cookies_enabled"`
```

Comment:
Would it make sense to have a cookies section instead?

Reply:
What would that look like?

Reply:
```
cookies:
    enabled: true
```

Reply:
oh. Not sure. I guess maybe cookies might have more options. I'll do it.